contracts/protocol/modules/LimitOrder.sol

```diff
@@ -63,6 +63,18 @@ abstract contract LimitOrderManager is ClaimableAmountManager {
                     _order.reduceLimitOrderId - 1
                 );
             }
+        } else if (_order.reduceQuantity != 0) {
+            if (_isReduce == 1) {
+                _orders[_orderIdx].reduceQuantity = partialFilled;
+            } else if (_order.reduceLimitOrderId != 0) {
+                PositionLimitOrder.Data[] storage _reduceOrders = _getLimitOrderPointer(
```

Comment:
if it's a close position limit order, need to remove it in `limitOrders` and update the reduce quantity in `reduceLimitOrders`